frontend/src/components/ThankyouForApplying.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from "./ThankyouForApplying.module.css";
+
+import { Page } from "../components/index.ts";
+
+export function ThankyouForApplying() {
+  return (
+    <Page>
+      <h1 className={styles.title}>Thank you for Applying!</h1>
+
+      <p className={styles.message}>
+        Your application seems to be missing a necessary component. Please review the requirements
+        for each pathway{" "}
+        <a
+          href="https://ccidc.org/becoming-a-certified-interior-designer/"
+          className={styles.lightBlue}
+        >
+          here
+        </a>{" "}
+        and reapply when you have met the requirements for at least path one.{" "}
+      </p>
+
+      <div className={styles.centeredContainer}>
+        <a href="https://ccidc.org/">
```

Comment:
Lets use the button componenet. Pull from main and use the new "additionalStyles" tag to apply the necessary styles.

For the page rounting you can look at Hrithik's [CongratulationsPopup](https://github.com/TritonSE/CCIDC-Application-Overhaul/blob/main/frontend/src/components/CongratulationsPopupWindow.tsx#L63)